### ISSUE TITLE
[Feature Request] Better DNS Resolving over Tor

### ISSUE THREAD

**Issue by ghost** [subgraph/subgraph-os-issues#260]
DNS over Tor is pretty PITA and once I am connected it is DNS that often seems to be slowing me down whether I'm trying to do updates via apt-get or browsing with Tor Browser it is DNS that seems to slow down traffic more than the quality of relays etc

**Comment by dma** [comment]
Yeah, agreed, DNS resolutions fail far too often over Tor. I've heard newer (than what is in Stretch) Tor improves this. Worth a try.

**Comment by ghost** [comment]
How do we use a new tor given the potential apparmor issues? Upstream Tor does have according to changelogs a lot of performance and dns improvements but Debian won't see those anytime soon.
